### ISSUE TITLE
[Question] About the final exam coverage

### ISSUE THREAD

**Issue by 08kmc09** [kaist-cp/cs431#944]
![image](https://github.com/kaist-cp/cs431/assets/74726456/cee68624-910f-4064-b183-ce55160885c4)

In this slide, professor mentions that promises will not be the scope of the final exam. Can I kindly ask which slides are excluded? I believe the pp.102 ~ 110 are the excluded ones.

**Comment by kingdoctor123** [comment]
You're right. "Promise" mechanism in the promising semantics (p.102 ~ p.110) is excluded, but you should study all remaining parts of the promising semantics (e.g. multi-valued memory, views, ...). I'll post an announcement regarding final exam soon.

**Comment by GWS0428** [comment]
If I remember correctly, the professor said that the information on the slide is incorrect and the content in the following slides will be covered in the final exam since we have learned it in class. Did I misunderstand the professor's instructions?

**Comment by jeehoonkang** [comment]
It's included.

**Comment by 08kmc09** [comment]
@jeehoonkang Oh I see, thank you professor!
